tests/main/upgrade-from-2.15/task.yaml

```diff
@@ -3,6 +3,7 @@ summary: Ensure upgrades from snapd 2.15 work
 systems: [ubuntu-16.04-64]
 
 prepare: |
+    apt-tool checkpoint > installed.pkgs
```

Comment:
Nitpick: taking an argument with filename vs printing the output is inconsistent.